### PR TITLE
Waitlist Bans

### PR DESCRIPTION
**Pull request by xBytez**
Since plug.dj introduced waitlist bans, this also introduced new events and backend API endpoints.

I did this quick draft, please draft and merge if you approve @Burkes OR @Dazzuh .

Thank you!

PS: If you want any changes, please feel free to commit to my branch or comment here.

### CODE REVIEW

File: api/endpoints/booth_waitlistban.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+**WaitlistBanDuration**: The duration member can have the following values:
+
+**f**: User is banned permanently
+
+**l**: User is banned for a day
+
+**m**: User is banned for an hour
+
+**l**: User is banned for 15 minutes
```
Comment (Dazzuh) [inline api/endpoints/booth_waitlistban.md]: L has been used twice here, did you mean s?

**Review by Dazzuh — COMMENTED** [review]
Seems good other than the one small issue I commented on.

**Comment by xBytez** [comment]
@Dazzuh Ooh, nice catch. Should be fixed now.

**Review by Dazzuh — APPROVED** [review]

**Comment by xBytez** [comment]
Thank you <3

**Pull request merged**